includes/routes/attendee/list.php

```diff
@@ -43,14 +43,20 @@ public function handle( string $event_slug ): void {
 		if ( ! current_user_can( 'edit_translation_event_attendees', $event->id() ) ) {
 			$this->die_with_error( esc_html__( 'You do not have permission to edit this event\'s attendees.', 'gp-translation-events' ), 403 );
 		}
-		$attendees = array();
 		if ( gp_get( 'filter' ) && 'hosts' === gp_get( 'filter' ) ) {
 			$is_active_filter = true;
 			$attendees        = $this->attendee_repository->get_hosts( $event->id() );
 		} else {
 			$attendees = $this->attendee_repository->get_attendees( $event->id() );
 		}
 
-		$this->tmpl( 'events-attendees', get_defined_vars() );
+		$this->render(
```

Comment:
If you prefer, I can change the code so that we call `$this->tmpl()` instead of `$this->render()`.